docfx.json

```diff
@@ -289,6 +289,7 @@
                 "docs/windows-workflow-foundation/3*.md": "error-reference",
                 "docs/windows-workflow-foundation/4*.md": "error-reference",
                 "docs/windows-workflow-foundation/5*.md": "error-reference",
+                "docs/whats-new/**/*.md": "whats-new",
```

Comment:
@BillWagner I set this one at the folder-level since the files are auto-generated.